web/yarn.lock

```diff
@@ -7429,9 +7429,9 @@ d3-collection@1, d3-collection@^1.0.3:
   resolved "https://registry.npmjs.org/d3-collection/-/d3-collection-1.0.7.tgz"
   integrity sha512-ii0/r5f4sjKNTfh84Di+DpztYwqKhEyUlKoPrzUFfeSkWxjW49xU2QzO9qrPrNkpdI0XJkfzvmTu8V2Zylln6A==
 
-d3-color@1, d3-color@^1.0.3:
+d3-color@1, d3-color@^1.0.3, d3-color@^1.4.1:
   version "1.4.1"
```

Comment:
not sure why there is a dependabot warning- this is already using the patched version